CHANGELOG.md

```diff
@@ -40,6 +40,8 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 - Adjusted the ordering of users in the organization details page ([#50](https://github.com/sovity/authority-portal/issues/50))
```

Comment:
![image](https://github.com/sovity/authority-portal/assets/75735874/c1189440-b1b2-4c56-867c-7a3563ca9517)

Can we put the copyright and footer in the middle on the unauthenticated page, onboarding page etc.

Reply:
good idea, changed everywhere, looks good and text doesn't overlap